src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -28,66 +28,113 @@ namespace domain {
 namespace CoordMapsTimeDependent {
 /*!
  * \ingroup CoordMapsTimeDependentGroup
- * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
- *  where \f$X\f$ is the outer boundary.
+ * \brief Maps the radius \f$r\f$ as \f$r(t) = a(t)\rho + \left(b(t) - a(t)
+ * \frac{\rho^3} {R^2}\right)\f$ where \f$\rho\f$ is the radius of the source
+ * coordinates.
  *
- * The map scales the coordinates \f$\xi\f$ near the center by a factor
- * \f$a(t)\f$, while the coordinates near the outer boundary \f$X\f$, are scaled
+ * The map scales the radius \f$\rho\f$ in the source coordinates by a factor
+ * \f$a(t)\f$, while the coordinates near the outer boundary \f$R\f$, are scaled
  * by a factor \f$b(t)\f$. Here \f$a(t)\f$ and \f$b(t)\f$ are FunctionsOfTime.
  *
- * Currently, only a 1-D implementation.
+ * The mapped coordinates are given by:
+ *
+ * \f{align}{
+ * x^i = \left[a + (b-a) \frac{\rho^2}{R^2}\right] \xi^{\hat{i}}
+ *       \delta^i_{\hat{i}},
+ * \f}
+ *
+ * where \f$\xi^{\hat{i}}\f$ are the source coordinates.
+ *
+ * The inverse map is computed by solving the cubic equation:
+ *
+ * \f{align}{
+ * (b-a)\frac{\rho^3}{R^2} + a \rho - r = 0,
+ * \f}
+ *
+ * which is done by defining \f$q=\rho/R\f$, and solving
+ *
+ * \f{align}{
+ * q \left[(b-a) q^2 + a\right] - \frac{r}{R} = 0.
+ * \f}
+ *
+ * The source coordinates are obtained using:
+ *
+ * \f{align}{
+ * \xi^{\hat{i}} = \frac{qR}{r} x^i(t) \delta^{\hat{i}}_i
+ * \f}
+ *
+ * The Jacobian is given by:
+ *
+ * \f{align}{
+ * \frac{\partial x^i}{\partial \xi^{\hat{i}}}=
+ *    \left[a + (b-a) \frac{\rho^2}{R^2}\right] \delta^i_{\hat{i}}
+ *    + \frac{2 (b-a)}{R^2} \xi^{\hat{j}} \delta^i_{\hat{j}} \xi^{\hat{k}}
+ *      \delta_{\hat{k}\hat{i}}
+ * \f}
+ *
+ * The inverse Jacobian is given by:
+ *
+ * \f{align}{
```

Comment:
Thanks for adding the dox. Just a couple pieces I think might be tex typos:
 - I think you don't want the Kronecker delta in the inverse Jacobian to be divided by both of the denominator factors, only one of them
- I think the a/R^2 in the second denominator factor should be a R^2 (multiplication instead of division)
- I think you're missing the 2(b - a) numerator factor for the non-delta term.

Reply:
Bloody hell.. This is what I get for trying to make the documentation more concise -_- thanks!